product-service/serverless.yml

```diff
@@ -15,7 +15,16 @@ provider:
   stage: dev
   region: eu-west-1
 
+  environment:
+    PG_HOST: aws-node-db-instance.csgvm17alrq7.eu-west-1.rds.amazonaws.com
+    PG_PORT: 5432
+    PG_DATABASE: fake-db-name
+    PG_USERNAME: postgres
+    PG_PASSWORD: fakePassword
```

Comment:
Please, remove your env variables from repo and change password, it's very unsecure. Everyone can connect to your DB.

Reply:
It is not real password and database name, I will update it later to store this data in `.env`